client/src/components/CheckInCal.jsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import moment from 'moment';
+class CheckInCal extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.localClick = this.localClick.bind(this);
+    this.convertMonthToString = this.convertMonthToString.bind(this);
+    this.getSundays= this.getSundays.bind(this);
+    this.createMonth = this.createMonth.bind(this);
+  }
+
+  localClick(e) {
+    e.preventDefault();
+  }
+
+  convertMonthToString(month) {
+    const months = ['January', "February", "March", "April", "May", "June",  "July", "August", "September", "October", "November", "December"]
+    let current = months[(month - 1)];
+    return current;
+  }
+
+  getSundays(y, n) {
+    const date = new Date(y, 0, 1);
+    while (date.getDay() != 0) {
+      date.setDate(date.getDate() + 1);
+    }
+    let days = [];
+    while (date.getFullYear() == y) {
+      const m = date.getMonth();
+      const d = date.getDate();
+      days.push(
+        [(m < 10 ? m : m),
+        (d < 10 ? d : d)]
+      );
+      date.setDate(date.getDate() + 7);
+    }
+    return days;
+  };
+
+  createMonth(t, n) {
+    let s = 0;
+    let days = [];
+    if (n === 2) {
+      days = Array.from({length: 28}, (_, i) => i + 1);
+    }
+    let thirty = [4, 6, 9, 11];
+    let thirtyOne = [1, 3, 5, 7, 8, 10, 12];
+    if (thirty.indexOf(n) === -1) {
+      days = Array.from({length: 31}, (_, i) => i + 1);
+    } else {
+      days = Array.from({length: 30}, (_, i) => i + 1);
+    }
+    const prev = n - 1;
+    const next = n + 1;
+    let pre = [];
+    let post = [];
+    const walk = function() {
+
+    };
+    walk();
+    let m = 42 - days.length;
```

Comment:
`m` is a number here, then on line 66 its an array. try to use actual words for your variable and ensure they are distinct

Reply:
no excuse but got fatigued...will do ty